data.lua

```diff
@@ -30,7 +33,40 @@ data:extend(
             filename = "__OutpostPlanner1-1__/graphics/miner-blueprint.png",
             width = 32,
             height = 32
-        }
-        -- TODO: Add shortcut
+        },
+      	{
+      			type = "shortcut",
+      			name = "outpost-builder",
+      			localised_name = {"item-name.outpost-builder"},
+      			order = "b[blueprint]-i[outpost-builder]",
+      			associated_control_input = "outpost-builder",
+      			action = "spawn-item",
+      			item_to_spawn = "outpost-builder",

```

Comment:
This will just spawn a new planner item every time someone hits the button, right?

I'm hesitant to add that without also adding something that deletes the item when you're done using it, otherwise players' inventory will fill up with planners.

Reply:
The flag "only-in-cursor" for the selection tool prototype (mentioned above) should make it so that as soon as you clear the cursor in any way the planner will just disappear.